data/data-pipeline/data_pipeline/etl/score/config/datasets.yml

```diff
@@ -129,3 +129,15 @@ datasets:
         include_in_tiles: true
         include_in_downloadable_files: true
 
+  - long_name: "DOT Travel Disadvantage Index"
+    short_name: "DOT"
+    module_name: "travel_composite"
+    input_geoid_tract_field_name: "GEOID10_TRACT"
+    load_fields:
+      - short_name: "travel_burden"
+        df_field_name: "TRAVEL_BURDEN_FIELD_NAME"
+        long_name: "DOT Travel Access Disadvantage" 
+        field_type: float
+        include_in_tiles: true
+        include_in_downloadable_files: true
+        create_percentile: true
```

Comment:
As a note, mostly for @lucasmbrown-usds: this is an average of percentiles (which, trivially, redistributes the percentile rank from uniform). I've created a percentile out of the percentile (and this is what the basic analysis we did shows), but that's sort of confusing. Interested in your thoughts here.